tests/scripts/remote_monitoring_tests/helpers/utils.py

```diff
@@ -302,10 +306,10 @@ def get_datetime():
 
 
 def term_based_start_time(input_date_str, term):
-    duration = {"short_term": 1, "medium_term": 7, "long_term": 15}
+    duration = {"short_term": 0.5, "medium_term": 48, "long_term": 192}
```

Comment:
This should still remain 1 day, 7 days or 15 days for short term, medium and long term, as the min data points should be checked within this duration.

Reply:
Tests are failing as monitoring start time in the recommendations json is now changed. For short term, difference between end time & start time is 30 mins, for medium term 2 days and long term 8 days. Is this as expected?

```
    "monitoring_end_time": "2024-02-08T21:57:37.682Z",
      
                    "short_term": {
                      "duration_in_hours": 24.0,
                       "monitoring_start_time": "2024-02-08T21:27:37.682Z", 
                   
                    "medium_term": {
                      "duration_in_hours": 168.0,
                      "monitoring_start_time": "2024-02-06T21:57:37.682Z",
                      
                    "long_term": {
                      "duration_in_hours": 360.0,
                      "monitoring_start_time": "2024-01-31T21:57:37.682Z",
                      
```

Reply:
@msvinaykumar - Is the monitoring_start_time as expected?

Reply:
No let me fix

Reply:
Fixed now , Please chek